applications/Tools/air_mouse/manifest.yml

```diff
@@ -0,0 +1,16 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/ginkage/FlippAirMouse.git
+    commit_sha: e001a1f0f2e41e67de6b1932452715212e347110
+short_description: Flipper Air Mouse
+description: "@README.md"
+changelog: "@CHANGELOG.md"
+screenshots:
+  - screenshot1.png
+  - screenshot2.png
+  - screenshot3.png
+name: Air Mouse
```

Comment:
You can omit 'name', 'id' & 'version' from .yml, if they are specified in application.fam